scss/_settings_colors.scss

```diff
@@ -26,6 +26,10 @@ $color-caution: #f99b11 !default;
 $color-positive: #0e8620 !default;
 $color-information: #24598f !default;
 
+$hover-darken-percentage: 10%;
```

Comment:
Could you add comments here to describe how these values are meant to be used ("use within `darken($color, $hover-darken-percentage)`" - or something similar)? - it's not clear without context.

Reply:
sure